src/components/Tables/Table.tsx

```diff
@@ -122,12 +123,16 @@ export type FilterConfig<Filter> =
     | {
           primaryFiltersShown: false;
           additionalFiltersShown: false;
+          listChoiceButton: false;
```

Comment:
This is a generic component, can we avoid adding specific properties to it?

The fundamental thing we're adding to the table is a special filter which _must_ have a value. If that can be fit into `primary_filters`, great! If not, then can we add a generic (and optional) field for mandatory filters.